src/library/scala/Predef.scala

```diff
@@ -199,14 +201,21 @@ object Predef extends LowPriorityImplicits {
 
   // TODO undeprecated until Scala reflection becomes non-experimental
   // @deprecated("use scala.reflect.classTag[T] and scala.reflect.runtime.universe.typeTag[T] instead", "2.10.0")
-  def manifest[T](implicit m: Manifest[T])           = m
+  def manifest[T](implicit m: Manifest[T]): Manifest[T]          = m
   // TODO undeprecated until Scala reflection becomes non-experimental
   // @deprecated("this notion doesn't have a corresponding concept in 2.10, because scala.reflect.runtime.universe.TypeTag can capture arbitrary types. Use type tags instead of manifests, and there will be no need in opt manifests.", "2.10.0")
-  def optManifest[T](implicit m: OptManifest[T])     = m
+  def optManifest[T](implicit m: OptManifest[T]): OptManifest[T] = m
 
   // Minor variations on identity functions
 
-  /** @group utilities */
+  /**
+   * A method that always returns the same value of its input argument `x`.
+   * Which could be used as a function argument passed to `map` or `flatMap` methods.
+   * Learn more from [[https://en.wikipedia.org/wiki/Identity_function Identity_function]]
```

Comment:
I'd remove this too. The wikipedia page is a mathematical discussion of the identity function and doesn't really help newcomer programmers to understand why it would be useful.